packages/ws/src/ws/WebSocketShard.ts

```diff
@@ -186,7 +186,18 @@ export class WebSocketShard extends AsyncEventEmitter<WebSocketShardEventsMap> {
 			this.connection &&
 			(this.connection.readyState === WebSocket.OPEN || this.connection.readyState === WebSocket.CONNECTING)
 		) {
+			// No longer need to listen to messages
+			this.connection.removeAllListeners('message');
```

Comment:
Use .off

Reply:
Is there a good reason for this?

Reply:
Consistency

Reply:
Hm, I wonder if I have to do `.off('message', this.onMessage`) or `.off('message', this.onMessage.bind(this))` to match how I originally set up the listener. I'll mess with it